__tests__/materials/material.mutation.test.js

```diff
@@ -257,8 +258,8 @@ describe('test material mutations', () => {
 
     expect(updatedMaterial).toBeDefined();
     expect(updatedMaterial).toHaveProperty('name', [
-      { __typename: 'Language', lang: 'uk', value: 'Тест update' },
-      { __typename: 'Language', lang: 'en', value: 'Test update' },
+      { __typename: 'Language', lang: 'uk', value: 'Тест mu ta tion' },
```

Comment:
Please, try to compare returned data with the data from the `materialToUpdate` object to avoid duplicated data and reduce the chance of mismatch errors if someone changes the data inside `materialToUpdate`

Reply:
Please, do it for all fields in materials tests

Reply:
now it is impossible to do because we receive   __typenames in objects. and in  `materialToUpdate` there is no __typenames

Reply:
fixed

Reply:
fixed